LICENSES/README.md

```diff
@@ -31,7 +31,8 @@ The following exceptions apply:
 
  * some sources under src/udev/ are licensed under **GPL-2.0-or-later**,
    so all udev programs (`systemd-udevd`, `udevadm`, and the udev builtins
-   and test programs) are also distributed under **GPL-2.0-or-later**.
+   and test programs), except for `libudev`, are also distributed under
+   **GPL-2.0-or-later**. `libudev` is distributed under **LGPL-2.1-or-later**.
```

Comment:
I like the basic direction. But please make the sentence more simple. How about something like the following?
```
 * some sources under src/udev/ are licensed under **GPL-2.0-or-later**,
   so all udev programs (`systemd-udevd`, `udevadm`, and the udev builtins
   and test programs) are also distributed under **GPL-2.0-or-later**.
   Note, sources under src/libudev are licensed under **LGPL-2.1-or-later**,
   so libudev library is also distributed under **LGPL-2.1-or-later**.
```
That is, keep the original sentence as is, and append additional note about libudev.

Reply:
Thanks! I'm not attached to my proposed sentenses. I just wanted to make it explicit about `libudev`. Keeping the original sentense sounds like a good idea!

I'll update this pull request as suggested. Or, if the members wanted to make changes on it by themselves/yourselves outside this pull request, that's also fine for me.